cloud/jenkins/pgo_operator_eks_latest.groovy

```diff
@@ -24,13 +24,8 @@ void prepareNode() {
         echo \$(kubectl kuttl --version) is installed
 
         curl -sL https://github.com/eksctl-io/eksctl/releases/latest/download/eksctl_\$(uname -s)_amd64.tar.gz | sudo tar -C /usr/local/bin -xzf - && sudo chmod +x /usr/local/bin/eksctl
-
-        sudo yum install -y https://repo.percona.com/yum/percona-release-latest.noarch.rpm || true
-        sudo percona-release enable-only tools
     """
-}
 
-void prepareSources() {
```

Comment:
I can't understand why it was done :( we logically split two preparation stages not you suggest to have one big function...

Reply:
We used to have the `prepareSources()` function just for the sake of it. What do I mean? We have used to have 2 functions: `prepareNode()` and `prepareSources()` and then we just call them one by one, like this:

```
    stages {
        stage('Prepare node') {
            steps {
                prepareNode()
                prepareSources()
            }
        }
```

There is absolutely no reason for that. I just combined the 2 functions into one: `prepareNode()`.